libraries/provider_database_sql_server_user.rb

```diff
@@ -106,12 +106,22 @@ def action_alter_sys_roles
           Chef::Log.info("SQL Server Version: #{server_version.inspect}")
           db.execute('USE [master]').do
           @new_resource.sql_sys_roles.each do |sql_sys_role, role_action|
-            alter_statement = if server_version < '11.00.0000.00'
```

Comment:
I think you reverted the SQL Server changes that went in recently with this commit